deployment/ansible/roles/nyc-trees.app/templates/nginx-nyc-trees-app.conf.j2

```diff
@@ -19,10 +19,20 @@ server {
   access_log /var/log/nginx/nyc-trees-app.access.log logstash_json;
 
   location /static/ {
+    {% if ['packer'] | is_in(group_names) -%}
+    etag on;
+    expires max;
+    {% endif %}
+
     alias {{ app_static_root }};
   }
 
   location /media/ {
+    {% if ['packer'] | is_in(group_names) -%}
+    etag on;
+    expires 1h;
```

Comment:
This was just a conservative number relative to the aggressive `max`. I also think I went with something conservative because I don't have a good idea of what is going to be served out of here yet.